tls/test_tls_integrity.py

```diff
@@ -89,7 +89,11 @@ def tcp_flow_check(self, resp_len):
         sniffer = analyzer.AnalyzerTCPSegmentation(remote.tempesta, 'Tempesta',
                                                    timeout=3, ports=(443, 8000))
         sniffer.start()
-        sleep(0.001) # Just schedule to another thread.
+        # TODO #120: the sniffer thread may not start with lower timeout like
+        # 0.001, so we use longer timeout here. Instead we should check whether
+        # the tcpdump process is running and wait for it otherwise.
+        # See appropriate comments in remote.py and analyzer.py.
+        sleep(0.1)
```

Comment:
Seems like the timeout should be listed inside `sniffer.start()` procedure, because it's related to all the places, where the `analyzer` is used.

Reply:
I see only this place and `tcp_connection/test_connection_close.py`, which doesn't call `sleep()`.

Reply:
I mean the `sleep()` here looks like synchronisation point, to assure, that `tcpdump` was started before sample request was sent. So I have a question, why the wait cycle is required here, but not in `tcp_connection/test_connection_close.py`. If I got this right, both tests might suffer from `tcpdump` timeouts.

Reply:
Fair enough